From 1e1bd1b7ddb2b259a88fcdc3fdeea6fad16d3b11 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Tue, 9 Aug 2016 23:50:59 +0300
Subject: [PATCH] Fix "make clang_mingw-builtins-arm" with latest master

This does the same to clang_linux.mk as was done in SVN rev
258863, git mirror commit 20da67f8d3f8.
---
 make/platform/clang_linux.mk | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/make/platform/clang_linux.mk b/make/platform/clang_linux.mk
index 870209f..6dce2e1 100644
--- a/make/platform/clang_linux.mk
+++ b/make/platform/clang_linux.mk
@@ -49,16 +49,14 @@ endif
 
 # Build runtime libraries for i386.
 ifeq ($(call contains,$(SupportedArches),i386),true)
-Configs += builtins-i386 profile-i386
+Configs += builtins-i386
 Arch.builtins-i386 := i386
-Arch.profile-i386 := i386
 endif
 
 # Build runtime libraries for x86_64.
 ifeq ($(call contains,$(SupportedArches),x86_64),true)
-Configs += builtins-x86_64 profile-x86_64
+Configs += builtins-x86_64
 Arch.builtins-x86_64 := x86_64
-Arch.profile-x86_64 := x86_64
 endif
 
 endif
@@ -71,17 +69,9 @@ CFLAGS := -Wall -Werror -O3 -fomit-frame-pointer
 
 CFLAGS.builtins-i386 := $(CFLAGS) -m32
 CFLAGS.builtins-x86_64 := $(CFLAGS) -m64
-CFLAGS.profile-i386 := $(CFLAGS) -m32
-CFLAGS.profile-x86_64 := $(CFLAGS) -m64
 
 FUNCTIONS.builtins-i386 := $(CommonFunctions) $(ArchFunctions.i386)
 FUNCTIONS.builtins-x86_64 := $(CommonFunctions) $(ArchFunctions.x86_64)
-FUNCTIONS.profile-i386 := GCDAProfiling InstrProfiling InstrProfilingBuffer \
-                          InstrProfilingFile InstrProfilingPlatformOther \
-                          InstrProfilingRuntime InstrProfilingUtil \
-                          InstrProfilingWriter InstrProfilingValue \
-                          InstrProfilingMerge InstrProfilingMergeFile
-FUNCTIONS.profile-x86_64 := $(FUNCTIONS.profile-i386)
 
 # Always use optimized variants.
 OPTIMIZED := 1
-- 
2.7.4

